src/plugins/openInNewTabAction/openInNewTabAction.js

```diff
@@ -33,6 +33,6 @@ export default class OpenInNewTab {
   }
   invoke(objectPath, urlParams = undefined) {
     let url = objectPathToUrl(this._openmct, objectPath, urlParams);
-    window.open(url);
+    window.open(url, undefined, "noopener");
```

Comment:
`noopener` means that the new window/tab does not share the same event loop as the opening window. It makes debugging A LOT easier and probably gives a subtle performance boost.

Reply:
See http://hassansin.github.io/shared-event-loop-among-same-origin-windows